### DISCUSSION TITLE
Keybind for toggling sidebar

### DISCUSSION THREAD

**Discussion by spacey-sooty** [zen-browser/desktop#3190] [Ideas]
[Screencast from 2024-11-02 20-01-07.webm](https://github.com/user-attachments/assets/1e2d3ad3-a69f-4c8d-bf12-b6c38d226e42)
I'd like this behaviour as a keybind

**Comment by arkriny** [comment]
![Screenshot 2024-11-28 193644](https://github.com/user-attachments/assets/6594f3a5-df27-4bac-a5cc-0825bfe21742)

There it is.
